### DISCUSSION TITLE
Valheim server doesn't run after transferring the save from Windows machine to Linux server

### DISCUSSION THREAD

**Discussion by kallekoivisto** [GameServerManagers/LinuxGSM#3297] [General]
I've got Valheim server working without issues with a fresh new world, but when I tried to move my existing world from my Windows gaming PC to the Linux server, the server starts shortly but stops after some seconds.

**Steps I did:**
* Stopped the server
* Copied the _.db_ and _.fwl_ files from AppData\LocalLow\IronGate\Valheim\worlds (Windows) to ~/.config/unity3d/IronGate/Valheim/worlds (Linux)
* Edited the lgsm/config-lgsm/vhserver/vhserver.cfg file with the following content:

```
##################################
####### Instance Settings ########
##################################
# PLACE INSTANCE SETTINGS HERE
## These settings will apply to a specific instance.

#### Game Server Settings ####

## Predefined Parameters | https://docs.linuxgsm.com/configuration/start-parameters
servername="sokerihumala"
# Minimum password length is 5.
serverpassword="*****"
port="2456"
gameworld="sokerihumala"
public="1"

## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
startparameters="-name '${servername}' -password ${serverpassword} -port ${port} -world ${gameworld} -public ${public}"

```
* Then I just started the server with `./vhserver start`:
```
[  OK  ] Starting vhserver: sokerihumala
```
* Then I observed that when I ran `./vhserver details` immediately I get Status: ONLINE, but if I ran it again right after the server is already Status: OFFLINE. So it's only on for a few seconds.

I don't know what is going on and why the server doesn't stay up. I'm also unable to find any useful logs or at least anything showing any errors. Where should I look?

**Distro Details**

Distro:    Ubuntu 20.04.2 LTS
Arch:      x86_64
Kernel:    5.4.0-65-generic
Hostname:  ubsrv2
Uptime:    0d, 5h, 28m
tmux:      tmux 3.0a
glibc:     2.31

**Valheim Server Details**

Server name:      sokerihumala
App ID:           896660
Server IP:        0.0.0.0:2456
Internet IP:      xxx.xxx.xxx.xxx:2456
Server password:  xxxxx
Game world:       sokerihumala
Status:           OFFLINE

**vhserver Script Details**

Script name:            vhserver
LinuxGSM version:       v21.1.3
glibc required:         2.15
Discord alert:          off
Email alert:            off
IFTTT alert:            off
Mailgun (email) alert:  off
Pushbullet alert:       off
Pushover alert:         off
Rocketchat alert:       off
Slack alert:            off
Telegram alert:         off
Update on start:        off
User:                   vhserver
Location:               /mnt/md0/gameservers/valheim

**Comment by kallekoivisto** [comment]
Nevermind.. Got it working. For some reason my 5-letter password was invalid for that transferred world. The same password was fine on the original Windows machine and also in the fresh world on the Linux machine. I added characters and the server stays on now :)

**Comment by wldwalker** [comment]
Solved
